samcli/lib/cookiecutter/interactive_flow_creator.py

```diff
@@ -42,6 +42,14 @@ def create_flow(flow_definition_path: str, extra_context: Optional[Dict] = None)
                         "True": "key of the question to jump to if the user answered 'Yes'",
                         "False": "key of the question to jump to if the user answered 'Yes'",
                       }
+                      "default": "default_answer",
+                      "default_from_toml": {
+                        "toml_file": "path/to/toml/file.toml",
```

Comment:
Does this have to be provided?

Reply:
@hoffa yes, we write the ARNs of the bootstrapped resources in its own toml file instead of in the samconfig.toml one.

@aahung it should as we use [os.path.normpath](https://github.com/elbayaaa/aws-sam-cli/blob/interactive_flow_question_default_answer_from_toml/samcli/lib/cookiecutter/question.py#L147-L148)